Use local wayland-scanner instead of looking for the native one in the host system.
Since we don't plan to do cross-compilation and wayland-scanner is built from wayland subproject, we override this. 
---
 protocol/meson.build | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/protocol/meson.build b/protocol/meson.build
index c60f2b9a..76cfbf24 100644
--- a/protocol/meson.build
+++ b/protocol/meson.build
@@ -5,10 +5,8 @@ wayland_protos = dependency('wayland-protocols',
 )
 wl_protocol_dir = wayland_protos.get_variable('pkgdatadir')
 
-wayland_scanner_dep = dependency('wayland-scanner', native: true)
 wayland_scanner = find_program(
-	wayland_scanner_dep.get_variable('wayland_scanner'),
-	native: true,
+	'wayland-scanner'
 )
 
 protocols = {
-- 
2.39.2

